spark/ingestion/src/main/scala/feast/ingestion/sources/bq/BigQueryReader.scala

```diff
@@ -46,6 +46,6 @@ object BigQueryReader {
     reader
       .load(s"${source.project}.${source.dataset}.${source.table}")
       .filter(col(source.eventTimestampColumn) >= new Timestamp(start.getMillis))
-      .filter(col(source.eventTimestampColumn) < new Timestamp(end.getMillis))
+      .filter(col(source.eventTimestampColumn) <= new Timestamp(end.getMillis))
```

Comment:
Why this change? When I see `start` and `end` i'd generally expect the convention to be [inclusive,exclusive).